webpack.config.js

```diff
@@ -18,12 +25,14 @@ module.exports = {
   },
   output: {
     filename: 'bundle.js',
-    path: __dirname + '/build'
+    path: path.resolve(__dirname, 'build'),
+    publicPath: '/build/',
   },
   devServer: {
-    publicPath: '/build/',
-    open: true,
-    openPage: 'build/home.html'
+  static: {
+      directory: path.join(__dirname, '/'),
+    },
+    open: ['build/home.html']
```

Comment:
Works in combination with the change in package.json#L8 to automatically open the (default) browser to the URL: `http://localhost:8080/build/home.html`